python/cuml/benchmark/bench_helper_funcs.py

```diff
@@ -26,3 +26,28 @@ def fit(m, x, y=None):
 
 def fit_transform(m, x):
     m.fit_transform(x)
+
+
+def predict(m, x, y=None):
+    m.predict(x) if y is None else m.predict(x, y)
+
+
+def _fil_classification_set_up(m, data, arg={}):
+    from cuml.utils.import_utils import has_xgboost
+    if has_xgboost():
+        import xgboost as xgb
+    else:
+        raise ImportError("No XGBoost package found which is required for benchmarking FIL")
+    import os 
+
+    # use maximum 10000 rows to train the model 
```

Comment:
I don't think the current code supports passing train/test split as a parameter (the default is 0.1), but the user could control the number of samples to train the model. 
In case of FIL, I use the train dataset (max: 1e4 samples) to train the xgboost model, and use the same data (whole train dataset) to predict for getting the performance number. Later use the test dataset to get the accuracy number. Do you see any problem here? How many samples do you think is enough for training the model?

Reply:
That depends on the model, of course. However, I'd expect around 1e6-1e7 samples for training, and 1e6 samples for benchmarking prediction. 1e4 is definitely too few.

Reply:
why are 1e4 samples too few for training? I worry we'll make a benchmark that takes a super long time to prep even before we start measuring timing.